marklogic-data-hub/src/main/java/com/marklogic/hub/impl/EntityManagerImpl.java

```diff
@@ -193,6 +207,31 @@ private HubModuleManager getPropsMgr() {
         return entities;
     }
 
+    private List<String> getEntityNamesInDatabase() {
```

Comment:
You'll want to remove this too -- we don't want to check database state to decide what to do with the filesystem